Events/CancelTrait.php

```diff
@@ -24,19 +24,17 @@ trait CancelTrait
     /**
      * Returns canceled.
      *
-     * @return bool
+     * @return null|bool
```

Comment:
nullable boolean parameters are kind of strange. doesnt it make sense to just use false as default here?